custom_components/myskoda/climate.py

```diff
@@ -179,33 +190,116 @@ def __init__(self, coordinator: MySkodaDataUpdateCoordinator, vin: str) -> None:
         )
         ClimateEntity.__init__(self)
 
+        self._attr_supported_features = (
+            ClimateEntityFeature.TURN_ON | ClimateEntityFeature.TURN_OFF
+        )
+
+        if self.vehicle.has_capability(
+            CapabilityId.AUXILIARY_HEATING_TEMPERATURE_SETTING
+        ) or self.vehicle.has_capability(
+            CapabilityId.AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY
+        ):
+            self._attr_supported_features |= ClimateEntityFeature.TARGET_TEMPERATURE
+
```

Comment:
Have you tested this? I rolled back changes because defining `_attr_supported_features` didn't work for me in `__init__`

Reply:
yes, I have tested it and works as expected

So if there is no AUXILIARY_HEATING_TEMPERATURE_SETTING or AIR_CONDITIONING_HEATING_SOURCE_AUXILIARY capability, climate entity will be created without target temperature feature (only on/off)